src/components/Chat/PublishModal.tsx

```diff
@@ -153,136 +162,157 @@ export default function PublishModal({
   return (
     <Modal
       portalId="theme-main"
-      containerClassName="inline-block h-[747px] min-w-[550px] max-w-[1100px] p-6 group/modal"
+      containerClassName={classNames(
+        'group/modal inline-block h-[747px] min-w-full max-w-[1100px] !bg-layer-2 md:min-w-[550px]',
+        { 'w-full': files.length },
+      )}
       dataQa="publish-modal"
       isOpen={isOpen}
       onClose={onClose}
       initialFocus={nameInputRef}
+      hideClose
     >
-      <div className="flex h-full flex-col gap-2">
-        <h4 className="max-h-[50px] text-base font-semibold">
+      <div className="flex h-full flex-col divide-y divide-tertiary">
+        <h4 className=" min-h-[50px] p-4 text-base font-semibold">
           <span className="line-clamp-2 break-words">
             {`${t('Publication request for')}: ${entity.name.trim()}`}
           </span>
         </h4>
+        <div className="flex min-h-0 grow flex-col divide-y divide-tertiary overflow-y-auto md:flex-row md:divide-x md:divide-y-0">
+          <div className="flex w-full shrink grow flex-col divide-y divide-tertiary md:max-w-[550px] md:overflow-y-auto">
+            <section className="flex flex-col gap-3 p-4">
+              <h2>{t('General Info')}</h2>
+              <p className="text-secondary">
+                {t(
+                  'Your conversation will be visible to organization only after verification by the administrator',
+                )}
+              </p>
+
+              <div>
+                <label
+                  className="mb-1 flex text-xs text-secondary"
+                  htmlFor="requestName"
+                >
+                  {t(`${getPrefix(entity)} name`)}
```

Comment:
To use dynamic values in translations in the **i18next.t** we need to do it in this way

       ```suggestion
                  {t('{{prefix}} name', {prefix: getPrefix(entity)})}
```

https://www.i18next.com/translation-function/interpolation